tests/host_task/host_task_interop_api.cpp

```diff
@@ -0,0 +1,126 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide verification for sycl::interop_handle::get_native functions
+//  this test check interop API with OpenCL back-end only.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+
+#define TEST_NAME host_task_interop_api
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+
+class TEST_NAME : public sycl_cts::util::test_base {
+  /** return information about this test
+   */
+  void get_info(test_base::info& out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+#ifdef SYCL_BACKEND_OPENCL
+  cl_int call_opencl(cl_command_queue q, cl_mem mem, size_t size,
+                     size_t pattern) {
+    cl_event e{};
+    cl_int ret{clEnqueueFillBuffer(q, mem, &pattern, sizeof(size_t),
+                                   /*offset*/ 0, size * sizeof(size_t), 0,
+                                   nullptr, &e)};
+    if (ret == CL_SUCCESS) {
+      ret = clWaitForEvents(1, &e);
+    }
+    return ret;
+  }
+#endif  // SYCL_BACKEND_OPENCL
+
+  /** execute this test
+   */
+  void run(util::logger& log) override {
+#ifdef SYCL_BACKEND_OPENCL
+    for_type_and_vectors<check_buffer_ctors_for_type, sycl::cl_half>(
+        log, "sycl::cl_half");
+    try {
+      sycl::queue q{util::get_cts_object::queue()};
+      if (q.get_backend() != sycl::backend::opencl) {
+        log.note("Interop part is not supported on this backend type");
+        return;
+      }
+
+      // check get_native_queue
+      {
+        cl_command_queue cl_native_queue{nullptr};
+        q.submit([&](sycl::handler& cgh) {
+          cgh.host_task([=, &cl_native_queue](sycl::interop_handle ih) {
+            cl_native_queue = ih.get_native_queue();
+          });
+        });
+        q.wait_and_throw();
+
+        if (cl_native_queue != q.get())
```

Comment:
`sycl::queue::get`, as well as `sycl::device::get` and `sycl::context::get` have been removed in SYCL 2020, right? So maybe consider using `sycl::get_native` here, or construct an OpenCL context and queue first, followed by a `sycl::make_queue`?

Reply:
I've applied this comment in: https://github.com/KhronosGroup/SYCL-CTS/pull/129/commits/7e35e5655106323ad49392f42634c24a17f0ee1d